### PR TITLE
Rebase/element ios 1.8.16

### PR DESCRIPTION
**Pull request by phlniji**
#544

### CODE REVIEW

File: Riot/Assets/fr.lproj/Vector.strings [hunk 1]
```diff
@@ -568,16 +568,9 @@
 "settings_key_backup_info_not_valid" = "Cet appareil ne sauvegarde pas vos clés, mais vous avez une sauvegarde existante que vous pouvez restaurer et joindre.";
 "settings_key_backup_info_progress" = "Sauvegarde de %@ clés…";
 "settings_key_backup_info_progress_done" = "Toutes les clés ont été sauvegardées";
-<<<<<<< HEAD
-"settings_key_backup_info_not_trusted_from_verifiable_device_fix_action" = "Pour utiliser la récupération de messages sécurisée sur cet appareil, vérifiez %@ maintenant.";
 "settings_key_backup_info_not_trusted_fix_action" = "Pour utiliser la récupération de messages sécurisée sur cet appareil, saisissez votre phrase secrète ou votre clé de récupération maintenant.";
```
Comment (yostyle) [inline Riot/Assets/fr.lproj/Vector.strings:571]: Maybe add this update in Element iOS

File: Riot/Modules/Room/RoomViewController.m [hunk 2]
```diff
@@ -3047,174 +3039,6 @@ - (RoomTimelineCellIdentifier)cellIdentifierForCellData:(MXKCellData*)cellData a
...
-            else if (bubbleData.shouldHideSenderName)
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierOutgoingEmoteEncryptedWithoutSenderName : RoomTimelineCellIdentifierOutgoingEmoteWithoutSenderName;
-            }
-            else
-            {
-                cellIdentifier = showEncryptionBadge ? RoomTimelineCellIdentifierOutgoingEmoteEncrypted : RoomTimelineCellIdentifierOutgoingEmote;
-            }
-        }
-    }
```
Comment (yostyle) [inline Riot/Modules/Room/RoomViewController.m]: If you can, please add this code or comment it

Comment (phlniji) [reply Riot/Modules/Room/RoomViewController.m]: The code is still here (and commented) but in another place in the file (from line 2876 to 3041).
It's a presentation issue from GitHub.

File: Riot/Assets/fr.lproj/Vector.strings (hunk 1 above)
Comment (phlniji) [reply Riot/Assets/fr.lproj/Vector.strings:571]: Noted

File: Riot/Assets/fr.lproj/Vector.strings [hunk 3]
```diff
@@ -48,10 +48,10 @@
 "auth_phone_placeholder" = "Numéro de téléphone";
 "auth_repeat_password_placeholder" = "Répéter le mot de passe";
 "auth_repeat_new_password_placeholder" = "Confirmer le nouveau mot de passe";
-"auth_invalid_login_param" = "Nom d'utilisateur et/ou mot de passe incorrect";
-"auth_invalid_user_name" = "Les noms d'utilisateur ne peuvent contenir que des lettres, des chiffres, des points, des traits d'union ou des tirets bas";
-"auth_invalid_password" = "Mot de passe trop court (min 8)";
-"auth_invalid_email" = "L'adresse e-mail ne semble pas valide";
+"auth_invalid_login_param" = "Nom d’utilisateur et/ou mot de passe incorrect";
+"auth_invalid_user_name" = "Les noms d’utilisateur ne peuvent contenir que des lettres, des chiffres, des points, des traits d’union ou des tirets bas";
+"auth_invalid_password" = "Mot de passe trop court (min 6)";
```
[the remark this answers is not in the document]
Comment (phlniji) [reply Riot/Assets/fr.lproj/Vector.strings]: Fixed in last commit.

**Review by yostyle — APPROVED** [review]

**Review by giomfo — APPROVED** [review]

**Pull request merged**